src/main/java/net/sf/jabref/external/FindFullTextAction.java

```diff
@@ -93,7 +95,7 @@ public void update() {
             basePanel.output(Localization.lang("Finished downloading full text document"));
         }
         else {
-            String message = Localization.lang("Full text article download failed");
+            String message = Localization.lang("Full text document download failed");
```

Comment:
I changed this text to "This operation requires exactly one item to be selected." (new string) and also replaced "No entries selected." with "This operation requires one or more entries to be selected." (already existing).